dev-aws/kafka-shared/pubsub-examples.tf

```diff
@@ -31,3 +31,9 @@ module "example_process_batch_consumer" {
   consume_topics   = { (kafka_topic.pubsub_examples.name) : "example-consume-process-batch" }
   cert_common_name = "dev-enablement/example-consume-process-batch"
 }
+
+module "es_topic_indexer" {
+  source           = "../../modules/tls-app"
+  consume_topics   = { (kafka_topic.pubsub_examples.name) : "es-topic-indexer" }
```

Comment:
Can we change the name of the consumer group to `dev-enablement.es-topic-indexer`, please?

Reply:
[Done](https://github.com/utilitywarehouse/kafka-cluster-config/pull/142/commits/91503b7113d076988d5c2205cdf881a1cb5085b2)